#! /bin/sh -e
## 03_tor_manpage_in_section_8.dpatch by  <weasel@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Move the Tor manpage from section 1 to section 8.

if [ $# -lt 1 ]; then
    echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch} ${2:+-d $2}"

case "$1" in
    -patch) patch -p1 ${patch_opts} < $0;;
    -unpatch) patch -R -p1 ${patch_opts} < $0;;
    *)
        echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
        exit 1;;
esac

exit 0

@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' tor~/doc/tor-gencert.1.txt tor/doc/tor-gencert.1.txt
--- tor~/doc/tor-gencert.1.txt	2012-09-08 13:11:10.000000000 +0200
+++ tor/doc/tor-gencert.1.txt	2012-09-08 13:11:19.313812236 +0200
@@ -83,7 +83,7 @@
 
 SEE ALSO
 --------
-**tor**(1) +
+**tor**(8) +
 
 See also the "dir-spec.txt" file, distributed with Tor.
 
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' tor~/doc/tor-resolve.1.txt tor/doc/tor-resolve.1.txt
--- tor~/doc/tor-resolve.1.txt	2012-09-08 13:11:10.000000000 +0200
+++ tor/doc/tor-resolve.1.txt	2012-09-08 13:11:19.313812236 +0200
@@ -42,7 +42,7 @@
 
 SEE ALSO
 --------
-**tor**(1), **torify**(1). +
+**tor**(8), **torify**(1). +
 
 See doc/socks-extensions.txt in the Tor package for protocol details.
 
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' tor~/doc/tor.1.txt tor/doc/tor.1.txt
--- tor~/doc/tor.1.txt	2012-09-08 13:09:02.000000000 +0200
+++ tor/doc/tor.1.txt	2012-09-08 13:11:37.505595918 +0200
@@ -4,7 +4,7 @@
 // Learn asciidoc on http://www.methods.co.nz/asciidoc/userguide.html
 :man source:   Tor
 :man manual:   Tor Manual
-TOR(1)
+TOR(8)
 ======
 
 NAME
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' tor~/doc/torify.1.txt tor/doc/torify.1.txt
--- tor~/doc/torify.1.txt	2012-09-08 13:11:10.000000000 +0200
+++ tor/doc/torify.1.txt	2012-09-08 13:11:19.317812190 +0200
@@ -46,6 +46,6 @@
 
 SEE ALSO
 --------
-**tor**(1), **tor-resolve**(1), **torsocks**(1)
+**tor**(8), **tor-resolve**(1), **torsocks**(1)
 
 AUTHORS
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' tor~/doc/asciidoc-helper.sh tor/doc/asciidoc-helper.sh
--- tor~/doc/asciidoc-helper.sh	2012-09-19 11:10:16.000000000 +0200
+++ tor/doc/asciidoc-helper.sh	2012-09-22 11:09:56.920891269 +0200
@@ -45,7 +45,7 @@
       exit 1;
     fi
     if "$2" -f manpage $input; then
-      mv $base.1 $output;
+      mv $base.[18] $output;
     else
       cat<<EOF
 ==================================
